clusterloader2/cmd/clusterloader.go

```diff
@@ -101,6 +101,7 @@ func initFlags() {
 	flags.StringArrayVar(&testOverridePaths, "testoverrides", []string{}, "Paths to the config overrides file. The latter overrides take precedence over changes in former files.")
 	flags.StringVar(&testSuiteConfigPath, "testsuite", "", "Path to the test suite config file")
 	initClusterFlags()
+	test.InitModifierFlags()
```

Comment:
modifier.InitFlags()

[put the logic in modifier package]

Reply:
Moved to package modifier